source/server/config/network/ext_authz.cc

```diff
@@ -0,0 +1,69 @@
+#include "server/config/network/ext_authz.h"
+
+#include <chrono>
+#include <string>
+
+#include "envoy/config/filter/network/ext_authz/v2/ext_authz.pb.validate.h"
+#include "envoy/ext_authz/ext_authz.h"
+#include "envoy/network/connection.h"
+#include "envoy/registry/registry.h"
+
+#include "common/ext_authz/ext_authz_impl.h"
+#include "common/filter/ext_authz.h"
+#include "common/protobuf/utility.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+
+NetworkFilterFactoryCb ExtAuthzConfigFactory::createFilter(
+    const envoy::config::filter::network::ext_authz::v2::ExtAuthz& proto_config,
+    FactoryContext& context) {
+
+  ASSERT(!proto_config.stat_prefix().empty());
+  ASSERT(!proto_config.grpc_service().envoy_grpc().cluster_name().empty());
+
+  ExtAuthz::TcpFilter::ConfigSharedPtr ext_authz_config(
```

Comment:
This file is missing coverage, see https://27311-65214191-gh.circle-artifacts.com/0/build/envoy/generated/coverage/coverage.source_server_config_network_ext_authz.cc.html. Can you add tests? See https://github.com/envoyproxy/envoy/tree/master/test/server/config for examples.

Reply:
my bad...thanks for catching that. Let me address this.